src/containers/App/index.tsx

```diff
@@ -2,22 +2,25 @@ import { SearchProvider } from "../../hooks/useSearch"
 import { NetworkTabsProvider } from "../../hooks/useNetworkTabs"
 import { useDarkTheme } from "../../hooks/useTheme"
 import { Main } from "../Main"
+import { RequestViewSectionsProvider } from "@/hooks/useRequestViewSections"
 
 export const App = () => {
   const isDarkTheme = useDarkTheme()
 
   return (
     <NetworkTabsProvider>
-      <SearchProvider>
-        <div
-          className={isDarkTheme ? "dark" : ""}
-          data-color-scheme={isDarkTheme ? "dark" : "light"}
-        >
-          <div className="bg-gray-100 dark:bg-gray-800 text-gray-900 dark:text-white">
-            <Main />
+      <RequestViewSectionsProvider>
```

Comment:
I am not against Context API but we soon should discuss about how to do State Management on this project. If you see this comment @warrenday

Reply:
Agree, `zustand` is a solid option IMO.

Reply:
Happy with context here. I think a native solution is better than introducing a third-party lib, especially for simple use cases.